### PR TITLE
Duplicate azure-pipelines.yml

### PR DESCRIPTION
**Pull request by lonitra**
For pipeline migration purposes
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/11007)

### CODE REVIEW

File: azure-pipelines-PR.yml [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        -TsaInstanceURL $(_TsaInstanceURL)
+        -TsaProjectName $(_TsaProjectName)
+        -TsaNotificationEmail $(_TsaNotificationEmail)
+        -TsaCodebaseAdmin $(_TsaCodebaseAdmin)
+        -TsaBugAreaPath $(_TsaBugAreaPath)
+        -TsaIterationPath $(_TsaIterationPath)
+        -TsaRepositoryName $(_TsaRepositoryName)
+        -TsaCodebaseName $(_TsaCodebaseName)
+        -TsaOnboard $(_TsaOnboard)
+        -TsaPublish $(_TsaPublish)'
```
Comment (RussKie) [inline azure-pipelines-PR.yml:196]: This section is internal only, so it should have been removed from the public

File: azure-pipelines-PR.yml [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    variables:
+    - template: /eng/common/templates/variables/pool-providers.yml
+    jobs:
+    # Publish to Build Asset Registry in order to generate the ReleaseConfigs artifact.
+    - template: /eng/common/templates/job/publish-build-assets.yml
+      parameters:
+        publishUsingPipelines: true
+        pool:
+          name: $(DncEngInternalBuildPool)
+          demands: ImageOverride -equals windows.vs2022preview.amd64
```
Comment (RussKie) [inline azure-pipelines-PR.yml:170]: ditto

File: azure-pipelines-PR.yml [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+- ${{ if and(eq(variables['EnableLoc'], 'true'), ne(variables['isPublic'], 'true'), ne(variables['isPR'], 'true')) }}:
+  - stage: OneLocBuild
+    displayName: Publish localizable content to OneLocBuild
+    jobs:
+    - template: /eng/common/templates/job/onelocbuild.yml
+      parameters:
+        MirrorRepo: winforms
+        UseCheckedInLocProjectJson: true
+        LclSource: lclFilesfromPackage
+        LclPackageId: 'LCL-JUNO-PROD-WINFORMS'
```
Comment (RussKie) [inline azure-pipelines-PR.yml:154]: ditto

File: azure-pipelines-PR.yml [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+            /p:DotNetPublishUsingPipelines=true
+    - name: _OfficialBuildIdArgs
+      value: /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
+    # needed for signing
+    - name: _SignType
+      value: real
+    - name: _SignArgs
+      value: /p:DotNetSignType=$(_SignType) /p:TeamName=$(_TeamName) /p:Sign=$(_Sign)
+    - name: _Sign
+      value: true
```
Comment (RussKie) [inline azure-pipelines-PR.yml:115]: ditto

File: azure-pipelines-PR.yml [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+      value: ''
+  - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+    - group: DotNetBuilds storage account read tokens
+    - group: AzureDevOps-Artifact-Feeds-Pats
+    - name: _InternalRuntimeDownloadArgs
+      value: /p:DotNetRuntimeSourceFeed=https://dotnetbuilds.blob.core.windows.net/internal
+             /p:DotNetRuntimeSourceFeedKey=$(dotnetbuilds-internal-container-read-token-base64)
+
+  # Produce test-signed build for PR and Public builds
+  - ${{ if or(eq(variables['isPublic'], 'true'), eq(variables['isPR'], 'true')) }}:
```
Comment (RussKie) [inline azure-pipelines-PR.yml:87]: This line should be removed because this condition is always true

File: azure-pipelines-PR.yml [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+    - group: DotNet-Winforms-SDLValidation-Params
+  - ${{ if ne(variables['System.TeamProject'], 'internal') }}:
+    - name: _InternalRuntimeDownloadArgs
+      value: ''
+  - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+    - group: DotNetBuilds storage account read tokens
+    - group: AzureDevOps-Artifact-Feeds-Pats
+    - name: _InternalRuntimeDownloadArgs
+      value: /p:DotNetRuntimeSourceFeed=https://dotnetbuilds.blob.core.windows.net/internal
+             /p:DotNetRuntimeSourceFeedKey=$(dotnetbuilds-internal-container-read-token-base64)
```
Comment (RussKie) [inline azure-pipelines-PR.yml:84]: This should be removed

File: azure-pipelines-PR.yml [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+  - name: NativeToolsOnMachine
+    value: true
+  - name: isPublic
+    value: ${{ eq(variables['System.TeamProject'], 'public') }}
+  - name: isPR
+    value: ${{ eq(variables['Build.Reason'], 'PullRequest')}}
+
+  # used for post-build phases, internal builds only
+  - ${{ if and(ne(variables['isPublic'], 'true'), ne(variables['isPR'], 'true')) }}:
+    - group: DotNet-Winforms-SDLValidation-Params
```
Comment (RussKie) [inline azure-pipelines-PR.yml:75]: ditto

File: azure-pipelines-PR.yml [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+    value: DotNetCore
+  # clean the local repo on the build agents
+  - name: Build.Repository.Clean
+    value: true
+  - ${{ if or(startswith(variables['Build.SourceBranch'], 'refs/heads/release/'), startswith(variables['Build.SourceBranch'], 'refs/heads/internal/release/'), eq(variables['Build.Reason'], 'Manual')) }}:
+    - name: PostBuildSign
+      value: false
+  - ${{ else }}:
+    - name: PostBuildSign
+      value: true
```
Comment (RussKie) [inline azure-pipelines-PR.yml:63]: This should be unrolled

File: azure-pipelines-PR.yml (hunk 1 above)
Comment (lonitra) [reply azure-pipelines-PR.yml:196]: Thanks! I realized after completing the migration in main that these will need to be removed as well. I will take care of this as part of the migration for the release branches.

File: azure-pipelines-PR.yml (hunk 8 above)
Comment (lonitra) [reply azure-pipelines-PR.yml:63]: What do you mean by unrolled here?

**Pull request merged**